hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/dag/nodes/CompactNode.java

```diff
@@ -40,10 +40,11 @@ public CompactNode(Config config) {
    * if it has one.
    *
    * @param executionContext Execution context to run this compaction
+   * @param itrCount
    * @throws Exception  will be thrown if any error occurred.
    */
   @Override
-  public void execute(ExecutionContext executionContext) throws Exception {
+  public void execute(ExecutionContext executionContext, int itrCount) throws Exception {
```

Comment:
I'd like to use the config to drive the itrCount and not have this signature change please..

Reply:
yeah, this refers to current iteration count. Don't think we can get away with this.